plugins/command-manager/src/main/java/com/wazuh/commandmanager/jobscheduler/SearchThread.java

```diff
@@ -107,92 +97,54 @@ public static <T> T getNestedObject(Map<String, Object> map, String key, Class<T
     }
 
     /**
-     * Iterates over search results, updating their status field and submitting them to the
-     * destination
+     * Iterates over search results, updating their status field to {@link Status#FAILURE} if their
+     * delivery timestamps are older than the current time, and submitting them to the destination
```

Comment:
We are not submitting the orders now. Also, if an order's delivery timestamp has expired, wouldn't it be lower than the current timestamp?

For example:

```
Human date: Fri, 10 Jan 2025 13:32:25 GMT
Epoch timestamp: 1736515945


Human date: Fri, 10 Jan 2025 19:32:25 GMT
Epoch timestamp: 1736537545
```